locust/argument_parser.py

```diff
@@ -81,6 +81,27 @@ def find_locustfile(locustfile):
     # Implicit 'return None' if nothing was found
 
 
+def find_locustfiles(locustdir: str) -> List[str]:
+    """
+    /Users/michaelnester/git/locust/examples/locustfile.py
+
+    Ignores files that start with _
+    Ignores files named locust.py
+    """
+    file_paths = []
+    if not os.path.exists(locustdir):
+        sys.stderr.write(f"Could not find directory '{locustdir}'.")
+        sys.exit(1)
+
+    for r, d, f in os.walk(locustdir):
```

Comment:
Probably best to use longer variable names. I’d have to look up what `r` and `d` are :)